src/packages/v4/views/V4ProjectDashboard/hooks/useV4ProjectHeader.ts

```diff
@@ -24,20 +28,37 @@ export interface ProjectHeaderData {
 export const useV4ProjectHeader = (): ProjectHeaderData => {
   const router = useRouter()
   const { projectMetadata, projectId } = useProjectMetadataContext()
```

Comment:
Yeah I think we should just not use these shared contexts if we can avoid it, in V4 code. Lets try stick to the SDK-given contexts

Reply:
Yeah ok good idea